Command/StartCrawlerCommand.php

```diff
@@ -47,13 +48,25 @@ public function configure()
      */
     protected function execute(InputInterface $input, OutputInterface $output)
     {
+	$blacklist = [];
+	
+	if(count($input->getOption('blacklist')) > 0){	    
+	    $output->writeLn('<info>Blacklisted URLs:</info>');
+
+	    $blacklist = $input->getOption('blacklist');
+	    foreach($blacklist as $url){
+	        $output->writeLn('<comment>' . $url . '</comment>');	
+	    }
+	}
+
         $data = array(
             'uri' => $input->getArgument('url'),
-            'base_url' => $input->getArgument('url'),
+	    'base_url' => $input->getArgument('url'),
+	    'blacklist' => $blacklist
```

Comment:
I would always add a comma in the last element of an array (that is allowed), this will prevent errors when adding items in the future :)